packages/comparisons/tsdown.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "tsdown";
+
+export default defineConfig({
+	attw: {
+		profile: "esm-only",
+	},
```

Comment:
```suggestion
	attw: {
		enabled: "ci-only",
		profile: "esm-only",
	},
```

Changing `package.json` exports (and other things that can make attw unhappy) should happen pretty rarely. While building (since it's required to run tests) happens a lot locally. Disabling attw locally reduces build time from ~17s to ~10s on my machine. What do you think about enabling attw in ci only?

Reply:
> While building (since it's required to run tests)

Well, `tsc -b` is required, which tdown runs, but you actually don't need to go through tsdown for the tests.
Fair enough though, I'll do that 👍🏻